go/vt/tabletserver/tx_pool_test.go

```diff
@@ -187,7 +187,7 @@ func TestTxPoolBeginWithPoolConnectionError_Errno2006_Permanent(t *testing.T) {
 	// DBConn.Exec() will return the reconnect error as final error and not the
 	// initial connection error.
 	_, err = txPool.LocalBegin(context.Background())
-	if err == nil || !strings.Contains(err.Error(), "Lost connection to MySQL server at 'reading initial communication packet'") {
+	if err == nil || !strings.Contains(err.Error(), "Lost connection to MySQL server at '") || !strings.Contains(err.Error(), "(errno 2013)") {
```

Comment:
It would be cleaner to remove " at '", there are no error messages that have the same beginning, but without "at" or without the quote, right?